decidim-proposals/app/forms/decidim/proposals/proposal_form.rb

```diff
@@ -16,6 +16,10 @@ class ProposalForm < Decidim::Proposals::ProposalWizardCreateStepForm
       attribute :has_address, Boolean
       attribute :attachment, AttachmentForm
       attribute :suggested_hashtags, Array[String]
+      attribute :photos, Array[String]
```

Comment:
is this `Array[String]` correct? or it should be `Array[AttachmentForm]`, or `Array[Attachment]`..?

Reply:
I believe it's correct as it's an array of ids